app/src/components/v-image.vue

```diff
@@ -68,7 +68,14 @@ async function loadImage() {
 		const base64 = window.btoa(raw);
 		srcData.value = `data:${contentType};base64,${base64}`;
 	} catch (err) {
-		emit('error', err);
+		imgError.value = true;
+		var decodedString = String.fromCharCode.apply(null, new Uint8Array(err.response.data));
+		var obj = JSON.parse(decodedString);
```

Comment:
Please use `const` instead of `var`.
(Did you run `pnpm run lint` locally? I guess this would have resulted in a corresponding warning.)

Also could you explain what this is doing? I wonder if this is really necessary...